packages/stockflux-launcher/public/popupWindow.css

```diff
@@ -0,0 +1,31 @@
+body {
+    overflow: hidden;
+    padding: 4vh 4vw;
+}
+.popup{
+    color:var(--text-grey-lighter);    
```

Comment:
Can the popup have a border similar to the one used on the launcher?

Reply:
Added in a border to the CSS for popup windows.